src/client/components/search/Search.jsx

```diff
@@ -54,9 +61,11 @@ const Search = ({
   );
 };
 Search.propTypes = {
-  searchText: PropTypes.string.isRequired,
-  updateSearchTerm: PropTypes.func.isRequired,
+  clearSearchError: PropTypes.func.isRequired,
   clearSearchTerm: PropTypes.func.isRequired,
-  performSearch: PropTypes.func.isRequired
+  hasSearchError: PropTypes.bool.isRequired,
+  performSearch: PropTypes.func.isRequired,
+  searchText: PropTypes.string.isRequired,
+  updateSearchTerm: PropTypes.func.isRequired
```

Comment:
also "term" should be "text" in `clearSearchTerm` and `updateSearchTerm`. My bad again.  Final result if you follow this advice + the advice in my above comment:

`clearText` and `updateText`